dataland-frontend/src/components/generics/LegalPage.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <TheHeader :landingPage="landingPage" />
+  <main role="main">
+    <div class="w-12 flex justify-content-center">
+      <div class="lg:w-6 md:w-10 sm:w-10 text-left">
+        <slot />
+      </div>
+    </div>
+  </main>
+  <TheFooter :sections="landingPage?.sections" />
+</template>
+
+<script lang="ts">
+import TheFooter from "@/components/generics/TheNewFooter.vue";
```

Comment:
Please ensure that the Footer is actually at the bottom of the page and no white space is displayed after the footer. Compare for example the Imprint page which is so short that the Footer kind of hangs in the air:

![image](https://github.com/d-fine/Dataland/assets/35459207/e8b0c235-e9fb-4a4e-86e1-2b48551ce5b2)

I.e. include as much space under the text as necessary to make sure that at least one screen is filled by the overall content.

Reply:
Fixed